scripts/tools/silabs/FactoryDataProvider.py

```diff
@@ -48,6 +48,11 @@ class FactoryDataWriter:
     PRODUCT_LABEL_NVM3_KEY = "0x87210:"
     PRODUCT_URL_NVM3_KEY = "0x87211:"
     PART_NUMBER_NVM3_KEY = "0x87212:"
+    DIC_CA_CERT_OFFSET = "0x87213:"
+    DIC_DEV_CERT_OFFSET = "0x87214:"
+    DIC_DEV_KEY_OFFSET = "0x87215:"
```

Comment:
Is this key supposed to be a private key or a public key? Because this storage is "plain text" so this isn't secure if this is the private key